src/api/index.test.ts

```diff
@@ -6,9 +6,17 @@ import { getTestServer } from '@google-cloud/functions-framework/testing'
 import { getPositions } from '../runtime/getPositions'
 import { getShortcuts } from '../runtime/getShortcuts'
 import { Position } from '../types/positions'
-import './index'
 import { SerializedDecimalNumber } from '../types/numbers'
-
+import { NetworkId } from './networkId'
+import { getConfig } from './config'
+jest.mock('./config')
+jest.mocked(getConfig).mockReturnValue({
+  POSITION_IDS: [],
+  GET_TOKENS_INFO_URL: 'https://valoraapp.com/mock-endpoint',
+  GOOGLE_CLOUD_PROJECT: 'dev-project',
+  SHORTCUT_IDS: [],
+})
+import './index' // NOTE: there are side effects of importing this module-- loading config params from the environment in particular. so mocking configs MUST be done before importing.
```

Comment:
this effectful imports issue is not new to this PR, though I strongly recommend we fix it soon and avoid it in the future